### ISSUE TITLE
Unclear error message when incorrectly using (click) in template

### ISSUE THREAD

**Issue by Larisho** [angular/angular#10455]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report
[x ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
When using (click) incorrectly in the template, 
i.e
```
<button (click)="">
```
One receives a very unclear error message that is hard to diagnose.
Error message:
![captur](https://cloud.githubusercontent.com/assets/18038322/17340308/9d9e8400-58bd-11e6-9d02-86cb5c4974b2.PNG)

**Expected/desired behavior**
A better error message, for the sake of newcomers to angular 2

**Reproduction of the problem**
If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem via https://plnkr.co or similar (you can use this template as a starting point: http://plnkr.co/edit/tpl:AvJOMERrnz94ekVua0u5).

**What is the expected behavior?**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**
windows 7
* **Angular version:** 2.0.0-rc.4
 
* **Browser:** [Chrome 51] 
 
* **Language:** [TypeScript ]

**Comment by ericmartinezr** [comment]
Dup of https://github.com/angular/angular/issues/3754, fixed in https://github.com/angular/angular/pull/10391

**Comment by pkozlowski-opensource** [comment]
Yep, this was fixed just a day ago via  e73d051. Thnx @ericmartinezr !
